modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,89 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that after updating a custom object entry, triggered user notifications using Object Author and Current User terms are filled correctly for the To and Subject fields"
+	@priority = 5
+	test CanUseObjectAuthorAndCurrentUserTermsInUserNotificationOnCustomObject {
+		task ("Given: a custom object and a entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 180339",
+				objectName = "CustomObject180339",
+				pluralLabelName = "Custom Objects 180339");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject180339");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject180339");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject180339",
+				value = "Entry Test");
+		}
+
+		task ("And given: a user notification template with the applicable fields filled with the Author and Current User terms") {
+			UserNotificationAPI.createNotificationTemplate(
+				en_US = "Subject Test",
```

Comment:
In a user notification template you can use the others terms, like the "CURRENT_USER_" terms.